Alicerce/Sources/Logging/Formatters/LogItemLevelColorFormatter.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  LogItemLevelColorFormatter.swift
+//  Alicerce
+//
+//  Created by Meik Schutz on 04/04/17.
+//  Copyright © 2017 Mindera. All rights reserved.
+//
+
+protocol LogItemLevelColorFormatter {
```

Comment:
The idea was to separate the customisation of naming from the colors, so if a client wants to just add localisation of the naming just needs to implement/ customize the naming-related protocol while maintaining everything color related.